lib/reports.py

```diff
@@ -13,7 +13,9 @@
 VERI_EMOJI = {
     1: "\u2705",  # WHITE HEAVY CHECK MARK
     -1: "\u274c",  # CROSS MARK
-    0: "\u2139"  # INFORMATION SOURCE
+    0: "\u2139",  # INFORMATION SOURCE
+    2: "\u2b06", # UPVOTE
+    3: "\u2b07" # DOWNVOTE
```

Comment:
maybe change this to -2 so positive veris are positive, and negative are negative